PodcastGenerator/core/feed_generator.php

```diff
@@ -43,17 +44,17 @@ function generateRSS()
         </itunes:owner>
         <itunes:explicit>' . $config['explicit_podcast'] . '</itunes:explicit>
 		<itunes:category text="' . htmlspecialchars($config['itunes_category[0]']) . '"></itunes:category>'."\n";
-    if ($config['itunes_category[1]'] != '') {
+    if (isset($config['itunes_category[1]']) && $config['itunes_category[1]'] != '') {
```

Comment:
I think `!empty($config['itunes_category[1]'])` will kill two birds with one stone

Reply:
I don’t know if that will error when the var is undeclared? i’ll check that